### ISSUE TITLE
sap_vm_provision: amend DNS var prefix

### ISSUE THREAD

**Issue by sean-freeman** [sap-linuxlab/community.sap_infrastructure#56]
sap_vm_provision: amend DNS var prefix

```
sap_vm_provision_aws_*
sap_vm_provision_ibmcloud_private_dns_*
sap_vm_provision_msazure_private_dns_*
```

**Comment by marcelmamula** [comment]
@sean-freeman Coming here from https://github.com/sap-linuxlab/community.sap_infrastructure/pull/55
I think we should focus on private DNS as you mentioned and AWS should reflect same. Trying to work on public side of things would be bit risky and counterproductive.

```
sap_vm_provision_aws_private_dns_*
sap_vm_provision_ibmcloud_private_dns_*
sap_vm_provision_msazure_private_dns_*
sap_vm_provision_gcp_private_dns_*
```

**Comment by sean-freeman** [comment]
@marcelmamula Agreed, scoping the Ansible Role to exclusive use of Private DNS functionality will be safer for everyone involved; the end-user can always target a DNS Service that has the option for Public Domain resolving or Corporate DNS Resolver interconnectivity etc etc and perform such tasks manually.
